data/json/encounters/randenc_caravans.json

```diff
@@ -0,0 +1,70 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_RandEnc_Roadstop_add",
```

Comment:
Do we need any kind of tests setup to make sure we don't introduce regressions to these specific sets of eoc's?   Or are eoc's isolated enough that shouldn't be an issue?